### ISSUE TITLE
1Month Duration aggregate window no generating correctly.

### ISSUE THREAD

**Issue by MikeNeilson** [opendcs/opendcs#31]
Discovered while doing a Python aggregating algorithm of CWMS Total.~1Day.1Month data the month interval isn't getting generated correctly. Instead of the expected start and end of month the generated aggregate window is the current time to end of the current day.

**Comment by MikeNeilson** [comment]
You would think so but no. The particular example is slightly odd.

the calculation is "total from start of this month to today."

I will create a table to demonstrate: (assume a month is 3 days so I can type less)
| day | incoming | total |
| -----| ---------- | ------ |
| 1/1  | 0.1 | 0.1 |
| 1/2  | 0.2 | 0.3 |
| 1/3  | 0.0 | 0.3 |
| 2/1  | 0.4 | 0.4 |
| 2/2  | 0.1 | 0.5 |
| 2/3  | 0.2 | 0.7 |    

Basically the end goal is the windowing behavoir of a Normal Aggregate with the output behavior of running aggregate. 
We were doing this in a python algorithm and that side of things appeared to be working.
